upstairs/src/volume.rs

```diff
@@ -351,6 +358,9 @@ impl Volume {
                     );
                     showat += showstep;
                 }
+                // Pause just a bit between IOs so we don't starve the guest
+                // TODO: More benchmarking to find a good value here.
+                tokio::time::sleep(Duration::from_millis(250)).await;
```

Comment:
If there's a 10 GB volume (10737418240 bytes), divided by 262144 byte chunks, that's 40960 chunks. Multiply that by 0.250 and that's 10240 seconds, or 2.84 hours!

We shouldn't pause here. With the better job dependencies, I don't think guest IO will be blocked unless they happen to be interacting with the same blocks at the same time, and even then only for a moment.

Reply:
I'll test again with the dependencies work.  I did my initial testing before that work was done, and found that boot time was 4x as long if the scrubber is running while the system is booting.

I'm sure there is also more tuning to do with regards to the size of scrubber IOs.

And, a third note, the scrubber is not actually turned on yet in propolis, so we have a little time to play around with different values before committing to something (but not too much time)